cli/bpmetadata/tfconfig_test.go

```diff
@@ -408,6 +409,107 @@ func TestSortBlueprintRoles(t *testing.T) {
 	}
 }
 
+func TestParseBlueprintRoles_PerModuleMode_HappyPath(t *testing.T) {
+	const hclContent = `
+locals {
+  per_module_roles = {
```

Comment:
Where would test only roles be defined?

Reply:
`test_only` roles and services are not necessary in this case, currently the `int_test_roles` carries all the test required roles. We might wanna flatten the per module roles and concat them with the existing int_test_roles if needed.

In short, this per module roles not affecting the integration test roles configs, it is helping CFT to generate bpmetadata.

Reply:
Not sure why that matters. 

Updated the test case using a full iam.tf content.